cmd/oci-create-runtime-bundle/oci-create-runtime-bundle.1.md

```diff
@@ -26,6 +26,9 @@ runtime-spec-compatible `dest/config.json`.
 **--type**
   Type of the file to unpack. If unset, oci-create-runtime-bundle will try to auto-detect the type. One of "imageLayout,image"
 
+**--log-level**
+  Log level to create runtime bundle. To be one of panic, fatal, error, warn, info, or debug. If unset, the level is identical to info.
```

Comment:
I think we should follow runtime-tools and [make the default level `error`][1].  That covers the stuff that leads to non-zero exit codes.  Folks interested in additional detail can bump up `--log-level`, and more restricted default allows us to be more generous about the amount of warn- and info-level logging we put in (without making the default command too noisy).

[1]: https://github.com/opencontainers/runtime-tools/blame/a773a293e6a401b37450f8b751f63e73d5cfd4a2/man/oci-runtime-tool.1.md#L33